sources/api/migration/migrations/v1.11.0/aws-creds/src/main.rs

```diff
@@ -0,0 +1,35 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddSettingsMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new settings for AWS credential configuration.
+fn run() -> Result<()> {
+    if cfg!(variant_platform = "aws") {
+        migrate(AddSettingsMigration(&[
+            "settings.aws.config",
+            "settings.aws.credentials",
+            "settings.aws.profile",
+        ]))
```

Comment:
This cfg attribute needs a build script, like `pki-affected-services/build.rs`, in order for this to ever be set.